ng-websocket.js

```diff
@@ -239,8 +239,8 @@
             return me;
```

Comment:
Was working couple of years ago. It looks like the project is not supported, so close this PR if not needed.